<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QDialog" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Change Password</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/shopping.png</normaloff>:/icons/shopping.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font: 12pt &quot;MS Shell Dlg 2&quot;;
color:white;
}
QDialog{
background-color:rgb(54,54,54);
color:white;
}
QGroupBox{
border:1px solid grey;
color:white;
padding:25px,10px;
}
QLineEdit{
border:1px solid grey;
background-color:rgb(54,54,54);
border-radius:5px;
color:white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:1px solid grey;
border-radius:3px;
}
QToolTip{
background-color:black;
color:white;
border:1px solid grey;
}
QPushButton:hover{
border:2px solid white;
}
QPushButton:pressed{
border:3px solid orange;
}
QLineEdit{
color:white;
}</string>
     </property>
     <property name="title">
      <string>Change Password</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_username">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maxLength">
           <number>50</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="placeholderText">
           <string>Enter your username</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_old_password">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maxLength">
           <number>20</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter your old password</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Old password</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_new_pass">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maxLength">
           <number>20</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter your new password</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>New Password</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_newpass_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maxLength">
           <number>20</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Confirm your new password</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Confirm Password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cancel">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(139, 0, 0);</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_change">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 170, 127);</string>
          </property>
          <property name="text">
           <string>Change Password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ChangePasswordDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
